lib/rc_test.sh

```diff
@@ -138,29 +145,30 @@ test_had_new_artifacts() {
         "had_new_artifacts ${input}"
 }
 
-test_existing_artifacts_with_artifacts() {
-    actual="$(EXISTING_ARTIFACTS='{"app1":"v1.0.0"}' existing_artifacts myorg/cicd/production pulumi)"
-    assertEquals '{"app1":"v1.0.0"}' "${actual}"
-    assertEquals \
-        "pulumi config get artifacts --cwd=pulumi/cicd --stack=myorg/cicd/production" \
-        "$(recorded_commands)"
-}
-
-test_existing_artifacts_without_artifacts() {
-    actual="$(existing_artifacts myorg/cicd/production pulumi)"
-    assertEquals "{}" "${actual}"
-    assertEquals \
-        "pulumi config get artifacts --cwd=pulumi/cicd --stack=myorg/cicd/production" \
-        "$(recorded_commands)"
-}
+# test_existing_artifacts_with_artifacts() {
```

Comment:
are these tests intentionally commented out?

Reply:
Derp... those were for a function that no longer exists. Thanks for catching that; will fix!